distro/BUILD

```diff
@@ -108,16 +108,7 @@ genrule(
 bzl_library(
     name = "rules_pkg_lib",
     srcs = [
-        "//:version.bzl",
-        "//pkg:mappings.bzl",
-        "//pkg:package_variables.bzl",
-        "//pkg:path.bzl",
-        "//pkg:pkg.bzl",
-        "//pkg:providers.bzl",
-        "//pkg:rpm.bzl",
-        "//pkg/private:pkg_files.bzl",
-        "//pkg/private:util.bzl",
-        "//pkg/private/deb:deb.bzl",
+        "//:bzl_srcs",
```

Comment:
Why not just make a filegroup of the files listed originally here and reuse that.

I don't particularly like the public export of the .bzl file everywhere.  That is going to conflict with trying to lock out the private stuff when bzl file visibility becomes available soon (we hope).

Reply:
`.bzl` files in `private` packages will need to be included in this filegroup. It doesn't matter where the `.bzl` file is, if a top level file loads a anything from it, it will be required by `stardoc`. This doesn't alter anything about the `bzl-visibility` warnings, which is why I opened https://github.com/bazelbuild/rules_pkg/pull/467 originally, to try and catch visibility violations, but unfortunately, because loads can still occur in any package, it's required that all `.bzl` source files are included in the globl